main.tf copy

```diff
@@ -0,0 +1,36 @@
+# ApiGateway Table Resource Code
+module "api_gateway" {
+  source           = "./modules/Api_Gateway"
+  api_name         = var.api_name
```

Comment:
only name prefix should come from variable, not full name
same for all other resources